cdk/lib/support-workers.ts

```diff
@@ -429,6 +429,26 @@ export class SupportWorkers extends GuStack {
       treatMissingData: TreatMissingData.BREACHING,
       threshold: 0,
     }).node.addDependency(stateMachine);
+
+    new GuAlarm(this, `NoTierThreeAcquisitionInPeriodAlarm`, {
+      app,
+      actionsEnabled: isProd,
+      snsTopicName: `alarms-handler-topic-${this.stage}`,
+      alarmName: `URGENT 9-5 - ${this.stage} support-workers No successful Tier Three checkouts in 6h.`,
+      metric: new MathExpression({
+        label: "AllTierThreeConversions",
+        expression: "SUM([FILL(m1,0),FILL(m2,0),FILL(m3,0)])",
```

Comment:
Does SUM mean any of these three payment methods will indicate a succesful payment?

Reply:
Yeah exactly, at least one successful checkout using any payment method will stop the alarm from firing.